tests/lib/utils/utils-test.js

```diff
@@ -186,3 +186,38 @@ describe('parseArgs', () => {
     expect(parsedArgs).toEqual(['asd', 'qwe', 'zxc']);
   });
 });
+
+describe('getPropertyValue', () => {
```

Comment:
So I thought about that, but it's intended purpose is to evaluate ASTNode properties. As such, we *could* add some simple tests with simple objects, but ultimately we should be testing it with ASTNodes, otherwise we could be missing a silly regression.

Thoughts?